src/containers/quiz/question-comment/QuestionComment.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import SendIcon from '@mui/icons-material/Send'
+import AddCommentOutlinedIcon from '@mui/icons-material/AddCommentOutlined'
+import AvatarIcon from '~/components/avatar-icon/AvatarIcon'
+import { IconButton } from '~/design-system/components/icon-button/IconButton'
+import { createUrlPath } from '~/utils/helper-functions'
+import { Quiz } from '~/types'
+import styles from '~/containers/quiz/question-comment/QuestionComment.styles'
+
+interface QuestionCommentProps {
+  quiz: Quiz
+  onCommentSubmit: (comment: string) => void
+}
+
+const QuestionComment: React.FC<QuestionCommentProps> = ({
+  quiz,
+  onCommentSubmit
+}) => {
+  const [comment, setComment] = useState('')
+  const [isCommentOpen, setIsCommentOpen] = useState(false)
+  const [isSent, setIsSent] = useState(false)
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setComment(event.target.value)
+    setIsSent(false)
+  }
```

Comment:
```suggestion
  }
  
```

Add spaces between functions, please.

Reply:
Done.